<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>screen_rob</class>
 <widget class="QWidget" name="screen_rob">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:grey;
</string>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="butt_ChangeMode">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>500</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;
background-color:white;</string>
     </property>
     <property name="text">
      <string>Change mode</string>
     </property>
    </widget>
    <widget class="QLabel" name="LED_Gyr">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>500</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: green;
color: green;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Gyr">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>500</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:grey;
font: 75 16pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>Accelerometer </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>500</y>
       <width>241</width>
       <height>81</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="butt_Reset">
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Comic Sans MS&quot;;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="butt_Start">
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Comic Sans MS&quot;;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="butt_Stop">
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Comic Sans MS&quot;;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>380</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QCheckBox" name="chB_Izom">
      <property name="styleSheet">
       <string notr="true">background-color:grey;
font: 75 14pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Izometric cam.</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chB_Road">
      <property name="styleSheet">
       <string notr="true">background-color:grey;
font: 75 14pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Visible road</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chB_Follow">
      <property name="styleSheet">
       <string notr="true">background-color:grey;
font: 75 14pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Following mice </string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_Stoper">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(225, 225, 225);
font: 75 16pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Stoper</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLCDNumber" name="LCD_Stoper">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 0, 0);</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>20</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_LavaBar">
      <property name="styleSheet">
       <string notr="true">background-color:grey;
font: 75 16pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Time till eruption</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="bar_Lava">
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="LED_RobConn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>530</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: green;
color: green;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_RobConn">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>530</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:grey;
font: 75 16pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>Robot connect</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QOpenGLWidget" name="openGLWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>551</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
